File: backend/models/Token.js
const mongoose = require('mongoose');

const tokenSchema = new mongoose.Schema({
  token: {
    type: String,
    required: true,
  },
  userId: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'User',
    required: true,
  },
  createdAt: {
    type: Date,
    default: Date.now,
    expires: '1h', // Token vyprší po 1 hodině
  },
});

module.exports = mongoose.model('Token', tokenSchema);

File: Backend/Models/Token.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Backend.Models
{
    public class Token
    {
        [Key]
        public int Id { get; set; }
        
        [Required]
        public required string TokenString { get; set; }
        
        [Required]
        public required string UserId { get; set; }
        
        [Required]
        public DateTime CreatedAt { get; set; }
        
        [Required]
        public DateTime ExpiresAt { get; set; }
    }
}